# SHA1:2da4cc17c82e9ac0b3764abd55420a1e9d217c9d
#
# This file is autogenerated by pip-compile-multi
# To update, run:
#
#    pip-compile-multi
#
exceptiongroup==1.2.0
    # via pytest
iniconfig==2.0.0
    # via pytest
numpy==1.26.3
    # via pandas
packaging==23.2
    # via pytest
pandas==2.1.4
    # via -r basetest.in
pluggy==1.3.0
    # via pytest
pytest==7.4.4
    # via -r basetest.in
python-dateutil==2.8.2
    # via pandas
pytz==2023.3.post1
    # via pandas
six==1.16.0
    # via python-dateutil
tomli==2.0.1
    # via pytest
tzdata==2023.4
    # via pandas
